hazelcast/src/test/java/com/hazelcast/collection/impl/txnqueue/TransactionQueueTest.java

```diff
@@ -386,8 +386,7 @@ private void testIssue859And863(final HazelcastInstance instance1, final Hazelca
 
             interruptThreads(instance2Threads);
 
-            jointThreads(instance2Threads, 15);
-            jointThreads(instance1Threads, 15);
```

Comment:
We save 4*45 secs here with not joining on the not interrupted instance1 threads? Good finding. :100:

Reply:
@Donnerbart added the `assertJoinable` in the `finally` block.
@blazember all of these were detected by looking at the JFR recording of the test build and the biggest test latencies. I think we might have more of those around the test codebase.